packages/cli-core/API.md

```diff
@@ -35,6 +53,14 @@ export enum LogLevel {
     INFO = 1
 }
 
+
+// @public
+export class PackageManagerControllerFactory {
+    constructor(projectRoot: string);
+    getPackageManagerController(): NpmPackageManagerController | PnpmPackageManagerController | YarnClassicPackageManagerController | YarnModernPackageManagerController;

```

Comment:
```suggestion
    getPackageManagerController(): PackageManagerController;

```

The return type should be `type PackageManagerController`.
This will allow to hide (i.e. not export) `XPackageManagerController` and `PackageManagerControllerBase`

Reply:
Done.